nbqa/__main__.py

```diff
@@ -504,6 +434,8 @@ def _find_config_file(
     ----------
     command
         Third-party tool being run.
+    project_root
+        Root of repository, where .git / .hg / .nbqa.ini file is.
```

Comment:
:+1: looks like I forgot this...sooner or later we should perhaps include darglint so we don't miss this kind of thing